packages/web3-react/src/components/AlephiumConnect.tsx

```diff
@@ -110,8 +72,8 @@ export const AlephiumConnectProvider: React.FC<AlephiumConnectProviderProps> = (
     setOpen,
     route,
     setRoute,
-    connector,
-    setConnector,
+    connectorId,
+    setConnectorId,
```

Comment:
It was a bit confusing calling this variable `connector` while its value is just the `id` property of the `Connector` type. I think it's clearer this way!